dist/marionette.gridview.js

```diff
@@ -0,0 +1,237 @@
+(function(root, factory) {
+
+    if (typeof define === 'function' && define.amd) {
+        define(['underscore', 'backbone', 'backbone.marionette'], function(_, Backbone, Marionette) {
+            return factory(root, _, Backbone, Marionette);
+        });
+    } else {
+        root.Marionette.GridView = factory(root, root._, root.Backbone, root.Marionette);
+    }
+
+}(this, function(root, _, Backbone, Marionette) {
+    'use strict';
+  var GridView = {};
+  var DEFAULT_WIDGET_VIEW = 'WidgetView',
+      DEFAULT_WIDGET_NAME = 'No Name';
+  
+  GridView.Widget = Backbone.Model.extend({
+    defaults: {
+      viewType: DEFAULT_WIDGET_VIEW,
+      name:     DEFAULT_WIDGET_NAME,
+      x:        0,
+      y:        0,
+      width:    0,
+      height:   0,
+      widgetId: 0
+    },
+  
+    getGridstackAttributes: function() {
+      return {
+        id:     this.get('widgetId'),
+        x:      this.get('x'),
+        y:      this.get('y'),
+        width:  this.get('width'),
+        height: this.get('height'),
+        el:     '<div class="grid-stack-item"><div id="' + this.get('widgetId') + '" class="grid-stack-item-content"></div></div>'
+      };
+    },
+  
+    isDefaultView: function() {
+      return this.get('viewType') === this.getDefaultView();
+    },
+  
+    getDefaultView: function() {
+      return DEFAULT_WIDGET_VIEW;
+    }
+  
+  });
+  
+  GridView.WidgetList = Backbone.Collection.extend({
+    model: GridView.Widget
+  });
+  
+  GridView.WidgetView = Marionette.ItemView.extend({
+    template: _.template('<div class="some-widget"><p>default view</p></div>'),
+  
+    modelEvents: {
```

Comment:
Better to create a function to initialize model events instead of using the hash map because if someone override it, you lose your events define here.